server/views/components/goal-summary-card/_goalSummaryCardContent-goalInStatus-ARCHIVED.njk

```diff
@@ -1,23 +1,5 @@
-<div class="govuk-summary-card__content" data-qa="archived-goal">
-  {% include './_goalSteps.njk' %}
-
-  {% if params.goal.notesByType.GOAL.length > 0 %}
-    {% set goalNote = params.goal.notesByType.GOAL[0] %}
-    <details class="govuk-details app-notes-expander" data-qa="goal-notes-expander">
-      <summary class="govuk-details__summary">
-            <span class="govuk-details__summary-text">
-              View note<span class="govuk-visually-hidden"> about this goal</span>
-            </span>
-      </summary>
-      <div class="govuk-details__text">
-        <span class="app-u-multiline-text">{{ goalNote.content }}</span>
-      </div>
-    </details>
-  {% endif %}
-
-  <p class="govuk-hint govuk-!-font-size-16" data-qa="goal-archived-hint">
-    Archived on {{ params.goal.updatedAt | formatDate('D MMMM YYYY') }}<span class="govuk-!-display-none-print"> by {{ params.goal.updatedByDisplayName }}, {{ params.goal.updatedAtPrisonName }}</span>
-  </p>
-
-  <p class="govuk-hint govuk-!-font-size-16" data-qa="goal-archive-reason-hint">Reason: {{ params.goal.archiveReason | formatReasonToArchiveGoal }}{{ ' - ' +  params.goal.archiveReasonOther if params.goal.archiveReason === 'OTHER'}}</p>
-</div>
+{% if not featureToggles.archiveGoalNotesEnabled %}
```

Comment:
The new designs for rendering archived goals is dependent on the feature toggle. ie. the feature toggle is not enabled in all envs yet, so we need to make sure that for envs that have the feature toggle disabled we show the old goal summary card, and for envs that have the toggle enabled we show the new goal summary card.

The _V1 file is basically what this file was previously

The _V2 file is the new goal summary card for Archived goals, with the new layout etc